### DISCUSSION TITLE
Report_supply_status.php white screen, no pdf

### DISCUSSION THREAD

**Discussion by alex001x** [opendcim/openDCIM#1517] [Q&A] [answered]
Hi Team Opendcim,
When I try to obtain the supply report, I get a white screen and no PDF output
On the opendcim.org and dev.opendcim.org site it is an HTTP error 500 message
I didn't find a solution in searches on github.
Are there writing rights?
or something else ?
Thanks for your help...
Alexandre.
FR

**Comment by wilpig** [comment, marked as answer]
https://github.com/opendcim/openDCIM/commit/95d12027971c68adb5898a89e82d2a82415168c6

I've corrected the issue for now, but it is slated to be replaced.

**Comment by alex001x** [comment]
Thks ;)
